modules/ocm-overview-tab.adoc

```diff
@@ -10,12 +10,17 @@ The **Overview** tab provides information about how the cluster was configured:
 
 * **Cluster ID** is the unique identification for the created cluster. This ID can be used when issuing commands to the cluster from the command line.
 * **Domain prefix** is the prefix that is used throughout the cluster. The default value is the cluster's name.
-* **Type** shows the OpenShift version that the cluster is using.
+* **Type** shows the type of cluster, e.g. {rosa-classic-short}, {rosa-short}, or Dedicated.
```

Comment:
does the guide https://spaces.redhat.com/display/CCSENABLE/Avoid+these+words tell us not to use Latin abbreviations? maybe use "for example"